### ISSUE TITLE
[🐛 Bug]: Mismatch between __version__ and release version

### ISSUE THREAD

**Issue by Th3DarkOn3** [SeleniumHQ/selenium#13282]
### What happened?

After the versione checking of selenium on python terminal I noticed a mismatching of actual released versione and the visible version (released = 4.16 ;   __version__ = 4.15.2).
Please update the file py/selenium/__init__.py

![image](https://github.com/SeleniumHQ/selenium/assets/16852776/ca8b8038-10df-4983-84db-e79c283e2e84)

### How can we reproduce the issue?

```shell
Go to the python terminal an type:

import selenium

selenium.__version__
```

### Relevant log output

```shell
no logs available
```

### Operating System

Windows 11

### Selenium version

Python 4.16

### What are the browser(s) and version(s) where you see this issue?

not relevant

### What are the browser driver(s) and version(s) where you see this issue?

not relevant

**Comment by diemol** [comment]
We are now using a script to bump versions in different parts. Unfortunately, it missed a couple of files and hence the issue you see. We cannot fix this in 4.16.0, but in the linked commit we have fixed the script to avoid this in the future. Thank you for reporting this.

**Comment by titusfortner** [comment]
Oops, this is my fault; I now remember that when I did the release for 4.15 I had to manually fix something, but then didn't go back and update the script, and forgot for 4.16. Thanks for fixing, Diego.
